app/actions/local/user.ts

```diff
@@ -105,25 +105,39 @@ export const updateRecentCustomStatuses = async (serverUrl: string, customStatus
     });
 };
 
-export const updateUserPresence = async (serverUrl: string, userStatus: UserStatus) => {
-    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
-    if (!operator) {
+export const updateLocalUser = async (serverUrl: string, userId: string, userDetails: Partial<UserProfile> & { status?: string}) => {
```

Comment:
you don't need `userId` as is part of `userDetails`, just make sure the id is set before doing anything else